src/PIL/TiffTags.py

```diff
@@ -18,10 +18,18 @@
 ##
 from __future__ import annotations
 
-from collections import namedtuple
+from typing import NamedTuple
 
 
-class TagInfo(namedtuple("_TagInfo", "value name type length enum")):
+class _TagInfo(NamedTuple):
+    value: int | None
+    name: str
+    type: int | None
+    length: int | None
+    enum: dict[int, str] | None
```

Comment:
```suggestion
    enum: dict[str, int]
```
Oh, sorry, I was wrong - `enum` can't be `None`.
`enum or {}` is right there in the code I referenced.

https://github.com/python-pillow/Pillow/blob/58c4c757a2e969f52d23586e731571887cad2db0/src/PIL/TiffTags.py#L27-L28

But also, it is `str` is the key, not the value in the dictionary.